external/java/pom.xml

```diff
@@ -54,7 +54,7 @@
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
-            <version>3.4.0</version>
+            <version>3.5.1</version>
```

Comment:
Out of curiosity: why did you need this new version here?

Reply:
I think this is a side-effect of debugging protobuf issues. I can change it back, if you'd like.

Reply:
No - I was just wondering if there is a bug in 3.4.0 that you wanted to avoid. If possible, I think the latest version should be best. So keep it.